src/mscorlib/model.xml

```diff
@@ -8152,6 +8152,10 @@
       <Member MemberType="Property" Name="Local" />
       <Member MemberType="Property" Name="Utc" />
     </Type>
+    <Type Name="System.TimeZoneInfo+AdjustmentRule">
+      <!-- CreateAdjustmentRule doesn't have a public contract, but tests and other callers can use Reflection to invoke it, so ensure it is left in mscorlib. -->
```

Comment:
I need the method to be public because the tests (and possible external code) are calling the method like this following:

```C#
Object rule = TestLibrary.Utilities.ExecuteMethod(
  TZISupport.AdjustmentRuleType, 
  null, 
  "CreateAdjustmentRule", 
  BindingFlags.Static | BindingFlags.Public, 
  new Object[] { this.DateStart, this.DateEnd, this.DaylightDelta, this.DaylightTransitionStart.ToInternalTransitionTime(), this.DaylightTransitionEnd.ToInternalTransitionTime() });
```

If the method wasn't public, the specified BindingFlags wouldn't work.